test/animation.js

```diff
@@ -260,7 +263,8 @@ exports["Animation -- Servo"] = {
     var indices = this.animation.findIndices(progress);
     var val = this.animation.tweenedValue(indices, progress);
 
-    test.ok(Math.abs(val - 74.843 < 0.01));
+    test.ok(Math.abs(val - 74.843) < 0.01,
+      "Expected " + val + " to be within 0.01 of 74.843");
```

Comment:
I'm surprised that the original assertion here ever worked, due to the misplaced paren; turns out
```javascript
(Math.abs(true) === 1) == true;
(Math.abs(false) === 0) == false;
```
I figured it wouldn't hurt to leave a nicer assertion behind. :grin: